### PR TITLE
updateWith() must abort when dup shippingOption ids

### PR DESCRIPTION
**Pull request by marcoscaceres**
Test for https://github.com/w3c/payment-request/pull/596

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2017-09-01 04:48:16
Finished: 2017-09-01 04:53:46


<h2>Failing Jobs</h2>
<ul>

<li>safari:10.0</li>

<li>MicrosoftEdge:14.14393</li>

</ul>




View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/18068)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/270689269)

File: payment-request/PaymentRequestUpdateEvent/updateWith-duplicate-shipping-options-manual.https.html [hunk 1]
```diff
@@ -0,0 +1,84 @@
+<!doctype html>
+<meta charset="utf8">
+<link rel="help" href="">
+<title>
```
Comment (edenchuang) [inline payment-request/PaymentRequestUpdateEvent/updateWith-duplicate-shipping-options-manual.https.html]: no title for the test.

File: payment-request/PaymentRequestUpdateEvent/updateWith-duplicate-shipping-options-manual.https.html [hunk 2]
```diff
@@ -0,0 +1,84 @@
+<!doctype html>
+<meta charset="utf8">
+<link rel="help" href="">
```
Comment (edenchuang) [inline payment-request/PaymentRequestUpdateEvent/updateWith-duplicate-shipping-options-manual.https.html]: no link

File: payment-request/PaymentRequestUpdateEvent/updateWith-duplicate-shipping-options-manual.https.html [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+test(() => {
+  try {
+    const request = new PaymentRequest(validMethods, validDetails);
+  } catch (err) {
+    done();
+    throw err;
+  }
+}, "Must construct a PaymentRequest (smoke test)");
+
+function testFireEvents(button) {
```
Comment (edenchuang) [inline payment-request/PaymentRequestUpdateEvent/updateWith-duplicate-shipping-options-manual.https.html]: should be disable button here?

**Review by edenchuang — APPROVED** [review]
please add link and title for test. and disable the test button after it is clicked.

File: payment-request/PaymentRequestUpdateEvent/updateWith-duplicate-shipping-options-manual.https.html (hunk 3 above)
Comment (marcoscaceres) [reply payment-request/PaymentRequestUpdateEvent/updateWith-duplicate-shipping-options-manual.https.html]: yep, added.

**Pull request merged**